### PR TITLE
Add Song Nick to Guild Wrapper and add fix command

### PR DESCRIPTION
**Pull request by binaryoverload**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] Tested changes

## What is added
Fix command

## What is fixed
Added song nick to guild wrapper

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                }
+            }
+        }
+        if (guild.isSongnickEnabled()) {
+            Player player = FlareBot.getInstance().getMusicManager().getPlayer(guild.getGuildId());
+            if (player.getPlaylist().isEmpty()) {
+                guild.getGuild().getController().setNickname(guild.getGuild().getSelfMember(), null).queue();
+            } else if (player.getPlayingTrack() != null) {
+                String str = null;
+                if (player.getPlayingTrack() != null) {
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java]: Double null check isn't needed

File: src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                    }
+                }
+            }
+        }
+        if (guild.isSongnickEnabled()) {
+            Player player = FlareBot.getInstance().getMusicManager().getPlayer(guild.getGuildId());
+            if (player.getPlaylist().isEmpty()) {
+                guild.getGuild().getController().setNickname(guild.getGuild().getSelfMember(), null).queue();
+            } else if (player.getPlayingTrack() != null) {
+                String str = null;
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java]: Why is this null and then properly initialized literally 2 lines later?

Comment (binaryoverload) [inline src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java]: Don't ask, this was made by arsen. Will fix

File: src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java (hunk 1 above)
Comment (binaryoverload) [inline src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java]: Yup, My bad

File: src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.util.Iterator;
+
+public class FixCommand implements Command {
+
+
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/general/FixCommand.java]: Double space

**Pull request merged**